### PR TITLE
Fix tutorials (continuation of #1036)

### PR DESCRIPTION
**Pull request by djarecka**
will update and add suggestions from #1036

I'm removing all `*.rst` files that are not used anymore (e.g. tutorial_101; see also discussion [here](https://github.com/miykael/nipype_tutorial/issues/37))

### CODE REVIEW

File: doc/devel/matlab_interface_devel.rst [hunk 1]
```diff
@@ -4,127 +4,85 @@
...
+            self.inputs.script =  self._my_script()
+            results = super(MatlabCommand, self).run( **inputs)
+            stdout = results.runtime.stdout
+            # attach stdout to outputs to access matlab results
+            results.outputs.matlab_output = stdout
+            return results
 
+        def _list_outputs(self):
+            outputs = self._outputs().get()
+            return outputs
```
Comment (effigies) [inline doc/devel/matlab_interface_devel.rst]: Lines 29-79 have been superseded by line 82 and should be removed. https://387-59325098-gh.circle-artifacts.com/0/home/circleci/work/docs/devel/matlab_interface_devel.html#example-2

File: doc/devel/cmd_interface_devel.rst [hunk 2]
```diff
@@ -39,7 +39,7 @@ above example we have used the ``desc`` metadata which holds human readable
 description of the input. The ``mandatory`` flag forces Nipype to throw an
 exception if the input was not set. ``exists`` is a special flag that works only
 for ``File traits`` and checks if the provided file exists. More details can be
-found at :doc:`interface_specs`.
+found at :ref:`interface_specs`.
```
Comment (effigies) [inline doc/devel/cmd_interface_devel.rst]: This change doesn't do anything. Might as well revert to keep the diff clean, unless there's a good reason to prefer `:ref:`.

File: doc/users/interface_tutorial.rst [hunk 3]
```diff
@@ -10,7 +10,8 @@ Specifying input settings
 The nipype interface modules provide a Python interface to external
 packages like FSL_ and SPM_.  Within the module are a series of Python
 classes which wrap specific package functionality.  For example, in
-the fsl module, the class :class:`nipype.interfaces.fsl.BET` wraps the
+the :ref:`fsl <nipype.interfaces.fsl>` module, the class
+:class:`nipype.interfaces.fsl.BET>` wraps the
```
Comment (effigies) [inline doc/users/interface_tutorial.rst]: `>` should be removed. Looks like a bad merge.

File: examples/fmri_ants_openfmri.py [hunk 4]
```diff
@@ -150,11 +160,12 @@ def create_reg_workflow(name='registration'):
...
-    registration can be made significantly more accurate for cortical
-    structures by increasing the number of iterations
-    All parameters are set using the example from:
-    #https://github.com/stnava/ANTs/blob/master/Scripts/newAntsExample.sh
+
+        * All parameters are set using the example from: \
+          `newAntsExample.sh <https://github.com/stnava/ANTs/blob/master/Scripts/newAntsExample.sh>`_
+        * This is currently set to perform a very quick registration. However,\
+          the registration can be made significantly more accurate for cortical\
+          structures by increasing the number of iterations.
```
Comment (effigies) [inline examples/fmri_ants_openfmri.py]: This is part of a docstring. It doesn't look very good. https://387-59325098-gh.circle-artifacts.com/0/home/circleci/work/docs/users/examples/fmri_ants_openfmri.html

I'd say revert this block.

File: examples/fmri_ants_openfmri.py [hunk 5]
```diff
@@ -350,11 +370,12 @@ def create_fs_reg_workflow(name='registration'):
...
```
Comment (effigies) [inline examples/fmri_ants_openfmri.py]: And this one.

File: examples/rsfmri_vol_surface_preprocessing_nipy.py [hunk 6]
```diff
@@ -425,11 +443,12 @@ def create_reg_workflow(name='registration'):
...
-    registration can be made significantly more accurate for cortical
-    structures by increasing the number of iterations
-    All parameters are set using the example from:
-    #https://github.com/stnava/ANTs/blob/master/Scripts/newAntsExample.sh
+
+        * All parameters are set using the example from: \
+          `newAntsExample.sh <https://github.com/stnava/ANTs/blob/master/Scripts/newAntsExample.sh>`_
+        * This is currently set to perform a very quick registration. However,\
+          the registration can be made significantly more accurate for cortical\
+          structures by increasing the number of iterations.
```
Comment (effigies) [inline examples/rsfmri_vol_surface_preprocessing_nipy.py]: And this one.

File: examples/fmri_ants_openfmri.py [hunk 7]
```diff
@@ -244,7 +257,10 @@ def create_reg_workflow(name='registration'):
 
 
 def get_aparc_aseg(files):
-    """Return the aparc+aseg.mgz file"""
+    """
+    Return the aparc+aseg.mgz file
+    """
+
```
[the remark this answers is not in the document]
Comment (djarecka) [reply examples/fmri_ants_openfmri.py]: ok, reverted, but this file contains many similar docstrings

**Pull request merged**